cmd/asset-syncer/utils.go

```diff
@@ -871,3 +882,21 @@ func (f *fileImporter) fetchAndImportFiles(name string, repo Repo, cv models.Cha
 	// entry if digest has changed
 	return f.manager.insertFiles(chartID, chartFiles)
 }
+
+// Check if two URL strings are in the same domain.
+// Return true if so, and false otherwise or when an error occurs
+func compareURLDomains(url1Str, url2Str string) bool {
```

Comment:
Slight preference for a name that indicates the boolean nature of this, such as `isURLDomainEqual` or `hostsEqual`.